### PR TITLE
TAG-3045 - fix multiple select focus loose

### PR DESCRIPTION
**Pull request by AndCrow**

### CODE REVIEW

File: components/angular-tomitribe-select/src/tomitribe-select.ts [hunk 1]
```diff
@@ -103,22 +103,33 @@ module tomitribe_select {
         function link(scope, element, attrs, uiSelect) {
             scope.openOnFocusDelay = angular.isDefined(attrs.openOnFocusDelay) ? parseInt(attrs.openOnFocusDelay) : 0;
             scope.openOnFocusDelayOnce = angular.isDefined(attrs.openOnFocusDelayOnce) ? attrs.openOnFocusDelayOnce : false;
+            const multiple = angular.isDefined(attrs.multiple);
```
Comment (Dexmaster) [inline components/angular-tomitribe-select/src/tomitribe-select.ts]: you already have uiSelect.multiple

**Review by Dexmaster — CHANGES_REQUESTED** [review]
small change is needed

File: components/angular-tomitribe-select/src/tomitribe-select.ts (hunk 1 above)
Comment (AndCrow) [reply components/angular-tomitribe-select/src/tomitribe-select.ts]: Yep, i know about it. But it's always ```undefined```. Looks like a bug.

Comment (Dexmaster) [reply components/angular-tomitribe-select/src/tomitribe-select.ts]: image

**Pull request merged**